android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -679,6 +687,91 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
         }
     }
 
+    private fun handleSnowflakeSubmit(
+        url: String,
+        captive_portal_url: String,
+        hysteriaCert: String?
+    ) {
+
+        // borrowed from the list Tor Browser uses: https://gitlab.torproject.org/tpo/applications/tor-browser-build/-/merge_requests/617/diffs
+        // too many is a problem, both of these seem to work for us
+        var ice = "stun:stun.l.google.com:19302,stun:stun.sonetel.com:3478,stun:stun.voipgate.com:3478"
```

Comment:
should these be moved into the credentials.properties?

Reply:
yes, but as an optional parameter in the envoy url.  this is basically the default value.

Reply:
Those are defaults. Apparently I forgot to updated the comment when I added a third